server/util/feedUtil.ts

```diff
@@ -54,17 +55,33 @@ export const getTorrentUrlsFromFeedItem = (feedItem: FeedItem): Array<string> =>
   return [];
 };
 
+const execAsync = (...command: string[]) => {
+  const p = spawn(command[0], command.slice(1));
+  return new Promise((resolveFunc) => {
+    p.stdout.on("data", (x) => {
+      process.stdout.write(x.toString());
+    });
+    p.stderr.on("data", (x) => {
+      process.stderr.write(x.toString());
+    });
+    p.on("exit", (code) => {
+      resolveFunc(code);
+    });
+  });
+}
+
 export const getFeedItemsMatchingRules = (
   feedItems: Array<FeedItem>,
   rules: Array<Rule>,
 ): Array<PendingDownloadItems> => {
   return feedItems.reduce((matchedItems: Array<PendingDownloadItems>, feedItem) => {
-    rules.forEach((rule) => {
+    rules.forEach(async (rule) => {
```

Comment:
this won't work

Reply:
I dont know how I never encountered a bug w/ this. I updated the function to be async